azurerm/internal/services/portal/portal_tenant_configuration_resource.go

```diff
@@ -0,0 +1,121 @@
+package portal
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/portal/mgmt/2019-01-01-preview/portal"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/portal/parse"
+	azSchema "github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/tf/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/timeouts"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourcePortalTenantConfiguration() *schema.Resource {
```

Comment:
Can we switch this out for the pluginsdk wrapper package instead of direct?
```suggestion
func resourcePortalTenantConfiguration() *pluginsdk.Resource {
```
This will need changing throughout.

Reply:
Updated